app/controllers/search_controller.rb

```diff
@@ -36,9 +37,10 @@ def index
 
     # mix in searchresults from appdata, as the api can't search in summary and description atm
     if (!@search_project)
-      appdata_hits = @appdata[:apps].select { |a| (a[:summary].match(/#{Regexp.quote(@search_term)}/i) ||
+      appdata_hits = @appdata[:apps].select do |a|
+        (a[:summary].match(/#{Regexp.quote(@search_term)}/i) ||
           a[:name].match(/#{Regexp.quote(@search_term)}/i))
-      }.map { |a| a[:pkgname] }
+      end.map { |a| a[:pkgname] }
```

Comment:
According to [Ruby style guide](https://github.com/bbatsov/ruby-style-guide#single-line-blocks):

> Avoid do...end when chaining

I actually would avoid chaining because it is more difficult to read. :thinking: what about:

```ruby

appdata_hits = @appdata[:apps].select do |a|
                 (a[:summary].match(/#{Regexp.quote(@search_term)}/i) ||
                 a[:name].match(/#{Regexp.quote(@search_term)}/i))
               end

@packagenames = (@packagenames + appdata_hits.map { |a| a[:pkgname] })
```

and actually the last line can be written as:

``` ruby
@packagenames += appdata_hits.map { |a| a[:pkgname] }
```

maybe this should be in another commit because it is not related to the Rubocop changes (but it hurts my eyes :see_no_evil:)